packages/element/src/react-platform.native.js

```diff
@@ -1,50 +0,0 @@
-/**
- * External dependencies
- */
-import { AppRegistry } from 'react-native';
-import { omit } from 'lodash';
-
-/**
- * WordPress dependencies
- */
-import { applyFilters, doAction } from '@wordpress/hooks';
-
-/**
- * Internal dependencies
- */
-import { Component, cloneElement } from './react';
-
-const render = ( element, id ) => {
-	class App extends Component {
-		constructor() {
-			super( ...arguments );
-
-			const parentProps = omit( this.props || {}, [ 'rootTag' ] );
-
-			doAction( 'native.pre-render', parentProps );
-
-			this.filteredProps = applyFilters(
-				'native.block_editor_props',
-				parentProps
-			);
-		}
-
-		componentDidMount() {
-			doAction( 'native.render', this.filteredProps );
-		}
-
-		render() {
-			return cloneElement( element, this.filteredProps );
-		}
-	}
-
-	AppRegistry.registerComponent( id, () => App );
```

Comment:
App registry logic is now done here: https://github.com/WordPress/gutenberg/blob/6daef3a01e7c6fb0bea45e621ca2f2b2bc4048f7/packages/react-native-editor/src/index.js#L23-L63.